### ISSUE TITLE
Integrate with OSX Finder: Connect to Server

### ISSUE THREAD

**Issue by Kriechi** [osxfuse/sshfs#26]
**Problem:**
Mounting a directory with SSHFS requires the user to jump into a Terminal window and type out the commands to create a mount point, and then mount with SSHFS.
This does not feel right for GUI-spoiled OSX users. It would be much nicer to make use of the Finder integration to mount remote filesystems: "Connect to Server" used for SMB or NFS mounts.

**Possible Solution:**
I stumbled across this [blog post](http://techiesfromthecradle.blogspot.de/2012/10/macosx-fuse-ssh-finder-and-other-goodies.html), and the [linked project code](https://groups.google.com/group/osxfuse-group/attach/f9076e11560ce2f5/SSHFS%20NetFS%20Plugin.zip?part=0.1).

This makes use of a bundle for the NetFS Finder integration.

IMHO it would be really great if this could be included in the osxfuse-SSHFS repository, to provide proper support and ideally also ship it (include it) directly (to be installed with homebrew).

Or is it better to report this over at https://github.com/libfuse/sshfs?

**Comment by bfleischer** [comment]
I agree with you, it would be nice to be able to mount sshfs volumes using "Connect to Server". If I remember correctly NetFS plugins need to be signed by Apple nowadays.

Currently I have my hands full with FUSE. Anyone up for the job of creating a NetFS plugin?

**Comment by Next-Door-Tech** [comment]
Just going to politely bump this in hopes that some kind soul has the time and interest to work on it. 😁
